### DISCUSSION TITLE
How about implementing a form on the website, in order to create a profile?

### DISCUSSION THREAD

**Discussion by Vedant-Manjrekar** [EddieHubCommunity/LinkFree#5650] [Ideas]
Currently to make a profile on linkfree one needs to manually create and edit the json file. Instead we can build a form which will collect necessary data from the user and make the profile for them. This way even non-coders would also be able to make a profile on linkfree.

**Comment by harisdev-netizen** [comment]
Yes, implementing a form to collect necessary data from users and automatically generate the JSON file for their LinkFree profile would likely make it easier for non-coders to create a profile. It can help simplify the process and reduce the chances of errors in the manual editing of the JSON file.

**Comment by poorvijn** [comment]
how about we create a drag and drop sort of an interface? I'm not sure about how feasible this would be to implement, but we can provide a basic template, and the user can edit the template and drag and drop elements like new links, images, new tabs etc

**Comment by shoebilyas123** [comment]
Yes. I have been thinking the same. It will also reduce the work on the maintainer's part as well as the await period for the profile to get updated part to validate the JSON as that will be handed over to the backend.

**Comment by ChinmayMhatre** [comment]
Closing the discussion. LinkFree now has forms to edit the profiles 🎉

**Reply by eddiejaoude** [reply]
Thank you 👍
